src/base/idx/View.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+  import type { Config } from '@app/config';
+  import { Caip10Link } from '@ceramicnetwork/stream-caip10-link';
+  import { EthereumAuthProvider } from '@ceramicnetwork/blockchain-utils-linking';
+  import CeramicClient from '@ceramicnetwork/http-client';
+  import { ceramic } from '@app/config.json';
+
+  export let did: string;
+  export let caip10: string;
+  export let config: Config;
+  
+  onMount(async () => {
+    if (config.signer) {
+      const address = await config.signer.getAddress();
+      console.log(address);
+      console.log(did);
+      const ethAuthProvider = new EthereumAuthProvider(
+        // TODO: Using the window.ethereum provider here since the config provider is throwing error.
+        config.provider, address
+      );
+
+      const ceramicClient = new CeramicClient(ceramic.api);
```

Comment:
Should we add this to Config? Eg. `config.ceramic.client`

Reply:
I see it's already in Config actually, but under idx

Reply:
"View" would be used for `/idx/:did`

Reply:
Yeah, we could eventually do two separate properties